bin/assignUsersToPersonalizedDigestVariation.ts

```diff
@@ -6,8 +6,8 @@ import { parse } from 'csv-parse';
 (async (): Promise<void> => {
   const csvFilePath = process.argv[2];
   const variationArgument = process.argv[3];
-  const batch = +process.argv[4] || 10000;
-  const splitBy = process.argv[4] || ',';
+  const batch = +process.argv[4] || 100;
+  const splitBy = process.argv[5] || ',';
```

Comment:
Adjustments I made to script importing variations:
- query adjustment
- argv adjustment (typo)
- batch smaller (when more rows are in the table `IN` queries are much more expensive)